index.html

```diff
@@ -1,75 +1,103 @@
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+      <header>
+        <img id="mainBanner" src="img/lambda-black.png" alt="Lambda Logo" />
+        <nav>
+          <p><a href="index.html">Home</a></p>
+          <p><a href="about.html">About</a></p>
+          <p><a href="#">Products</a></p>
+          <p><a href="#">Blog</a></p>
+          <p><a href="#">Contact</a></p>
+        </nav>
+      </header>
+      <img id="mainBanner" src="img/jumbo.jpg" alt="Main Banner" />
+      <section class="top-content">
+        <div class="text-container">
+          <h2>The Future</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+        <div class="text-container">
+          <h2>The Past</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+      </section>
+
+      <section class="middle-content">
+        <h2>Why Did It Have To Be Boxes...</h2>
 
-        <section class="bottom-content">
+        <div class="boxes">
+          <div id="box1" class="box">Box 1</div>
+          <div id="box2" class="box">Box 2</div>
+          <div id="box3" class="box">Box 3</div>
```

Comment:
Loved the way you used "id" in order to color the boxes in a different color. You can also keep the "box" class and add an additional class to each box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```

Nonetheless, great job!